about.html

```diff
@@ -0,0 +1,165 @@
+                <div class="contact-us">
+              <p> Please contact us per Email for any further questions
+                about Youth Global Empowerment!</p>
+                <a><b> Globalempowerment.org </b></a>
+            </div>
+          </section>
+
+           <!-- logo start  -->
+
+    <section class="section">
+        <div class="container">
+          <h2 class="sec-title">Global Youth Empowerment Summit 2023 Logo</h2>
+          <div class="line3"></div>
+          <p class="text-center">
+            The logo of GWE Summit 2022 was decided through the logo
+            competition from 8. June to 7. July.
+          </p>
+          <div class="logo-desc">
+            <img src="images/LogoMakr-3POqWB.png" alt="logo info">
+          </div>
+        </div>
+    </section>
+
+     <!-- Past Summits  -->
+     <section class="section1">
+        <div class="container">
+          <h2 class="sec-title">See the past Global Summit</h2>
+          <div class="line3"></div>
+          <p class="text-center">
+            Take a look at the last two Global Youth Empowerment Summit which took place
+            in Harare and in New York.
+          </p>
+          <div class="history">
+            <div class="book">
+          <div class="image-overlay">
+              <img src="images/history1.jpg" class="hi-img" alt="bootcamp" />
+              <div class="info">
+                <h3>2021</h3>
+                <p class="text-white">
+                    Global Youth Empowerment Summit 2021 in Harare, Zimbabwe
+                </p>
+            </div>
+              </div>
+            </div>
+  
+            <div class="book">
+          <div class="image-overlay">
+            <img src="images/history2.png" class="hi-img" alt="bootcamp" />
+              <div class="info">
+                <h3>2022</h3>
+                <p class="text-white">
+                    Global Youth Empowerment Summit 2022 in New York, USA
+                </p>
+            </div>
+          </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+<!-- partner section start -->
+    
+<section id="sponsors">
+    <div class="foot-cont">
+        <div class="foot-head">
+            <h1>Partners</h1>
+                <div class="line3"></div>
+            <div class="foot-box">
+                <img src="images/coca-cola-logo-svgrepo-com.svg" alt="partner1" class="part">
+                <img src="images/instagram-logo-svgrepo-com.svg" alt="partner2" class="part">
+                <img src="images/forbes-logo-svgrepo-com.svg" alt="partner3" class="part">
+                <img src="images/google-play-download-android-app-logo-svgrepo-com.svg" alt="partner4" class="part">
+                <img src="images/heineken-14-logo-svgrepo-com.svg" alt="partner5" class="part">
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
in both the home and about page please remove the padding added to the top of the partner section to improve your pages UX

your design 
![image](https://user-images.githubusercontent.com/60612329/217680685-512dc860-5c7b-446a-9e0c-61e34bf99229.png)

required design
![image](https://user-images.githubusercontent.com/60612329/217680736-043ed5cd-3d7d-4685-b925-74bea12c80d5.png)